addon/components/new-view.js

```diff
@@ -11,5 +16,33 @@ export default ActivitiesBaseNewView.extend({
 
    this.set('groups', []);
    this.set('parents', []);
+ },
+
+ actions: {
+   save() {
+     /* jshint ignore:start */
+     let save = this._super.bind(this, ...arguments);
+
+      let attrs = {
```

Comment:
Instead of `...attrs` later on, you can do:
```javascript
let attrs = Object.assign({}, {
        title: this.get('model.title'),
        description: this.get('model.description'),
        groups: this.get('model.groups'),
        parents: this.get('model.parents'),
        product_owner: this.get('session.user')
});
```